### PR TITLE
Implement award action and motion copy in event list

### LINKED ISSUE #2702: Award reputation events wiring

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Integrated award action and motion with the copies for the action and event list. The action should now appear as expected and not with the copy of smite or missing data in the copy.

This should be the last feature PR for `Award` before the feature is ready to be merged and tested on QA, just like `Smite`.

Resolves #2702

### CODE REVIEW

**Review by chinins — APPROVED** [review]
Good to go!

**Review by danbr — COMMENTED** [review]
Generally the copy looks good for me, but just a questions of pluralisation of 'pts' for  a single reputation point?

Screenshot 2022-01-20 at 11 26 39


Screenshot 2022-01-20 at 11 16 16


Screenshot 2022-01-20 at 11 17 45

Also, i dont know if this should be covered here but there is a bug when entering amount of points to award/deduct. The input is limited to 2 decimal places, even when clicking on 'max'

Screenshot 2022-01-20 at 11 22 18

File: src/modules/dashboard/transformers/transformers.ts [hunk 1]
```diff
@@ -253,9 +254,29 @@ export const getActionsListData = (
...
                 name,
               } = unformattedAction;
-              const actionEvent = Object.entries(ACTIONS_EVENTS).find((el) =>
-                el[1]?.includes(name.split('(')[0]),
-              );
+
+              let actionEvent;
+
+              if (
+                name.split('(')[0] ===
```
Comment (alicjakujawa) [inline src/modules/dashboard/transformers/transformers.ts:261]: I think we have helper for it (formatEventName)

File: src/modules/dashboard/transformers/transformers.ts [hunk 2]
```diff
@@ -253,9 +254,29 @@ export const getActionsListData = (
...
+                actionEvent = Object.entries(ACTIONS_EVENTS).find(
+                  (el) =>
+                    (isSmiteAction &&
+                      el[0] === ColonyActions.EmitDomainReputationPenalty) ||
+                    (!isSmiteAction &&
+                      el[0] === ColonyActions.EmitDomainReputationReward),
+                );
+              } else {
+                actionEvent = Object.entries(ACTIONS_EVENTS).find((el) =>
+                  el[1]?.includes(name.split('(')[0]),
```
Comment (alicjakujawa) [inline src/modules/dashboard/transformers/transformers.ts:276]: same here

**Review by alicjakujawa — APPROVED** [review]
small comments but approving

File: src/modules/dashboard/transformers/transformers.ts (hunk 1 above)
Comment (ArmandoGraterol) [reply src/modules/dashboard/transformers/transformers.ts:261]: Thank you! I didn't knew that

**Comment by ArmandoGraterol** [comment]
@danbr FYI, added a fix for your second point, however that bug may come back in QA when testing `Smite` as the bug was in the form, and both smite and award use basically the same form

**Pull request merged**